site/content/en/docs/Guides/Client SDKs/csharp.md

```diff
@@ -7,9 +7,11 @@ description: "This is the C# version of the Agones Game Server Client SDK."
 publishDate: 2020-05-28
 ---
 
+{{% feature expiryVersion="1.37.0" %}}

```

Comment:
The whole thing can be deleted - we have a NuGet package, so the warning doesn't make sense anymore.